waveformview/src/main/java/space/siy/waveformview/FixedWaveFormView.kt

```diff
@@ -406,17 +406,17 @@ class FixedWaveFormView(
     val bottomBars = Array(resampleData.size) { i ->
       val multiplier = i.toFloat()
       val x = (multiplier * blockWidth) + (multiplier * gapWidth)
-      val bottom = (height - height * bottomBlockScale) + verticalGap
-      var top = bottom + ((height - bottom) * resampleData[i] / maxAmplitude)
+      val paddedTop = (height - height * bottomBlockScale) + verticalGap
+      var top = paddedTop + ((height - paddedTop) * resampleData[i] / maxAmplitude)
       top = if (domeDrawEnabled) (top - domeRadius) else top
-      val paddedTop = if (top - bottom < MIN_HEIGHT) (bottom + MIN_HEIGHT) else top
+      val bottom = if (top - paddedTop < MIN_HEIGHT) (paddedTop + MIN_HEIGHT) else top
       RectF(x, paddedTop, x + blockWidth, bottom)
     }
 
     val bottomDomes = if (domeDrawEnabled) Array(resampleData.size) { i ->
       val bottomRect = bottomBars[i]
-      val bottom = bottomRect.bottom
-      val paddedTop = bottomRect.top
+      val bottom = bottomRect.top
+      val paddedTop = bottomRect.bottom
```

Comment:
make sure this doesn't break top domes. I think they drawArc so it should be fine.

Reply:
It doesn't break